.changeset/many-moments-juggle.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/container-loader": minor
+---
+
+Deprecated summarizeProtocolTree and it's corresponding duplicate ILoaderOptions definition from container layer.
+
+summarizeProtocolTree property in ILoaderOptions was added to test single-commit summaries during the initial implementation phase. The flag is no longer required or should be used, so marked it as deprecated. If a driver needs to enable or disable single-commit summaries, it can do so via IDocumentServicePolicies.
```

Comment:
Nit
```suggestion
summarizeProtocolTree property in ILoaderOptions was added to test single-commit summaries during the initial implementation phase. The flag is no longer required and should no longer be used, so marked it as deprecated. If a driver needs to enable or disable single-commit summaries, it can do so via IDocumentServicePolicies.
```